packages/api/constants.ts

```diff
@@ -0,0 +1,22 @@
+import {ApiOptions} from '@cennznet/api/types';
+
+export type NetworkConfig = {
+  options: ApiOptions,
+  defaultEndpoint: string
+};
+
+export type NetworkName = 'CUSTOM' | 'RIMU' | 'KAURI';
```

Comment:
I would suggest to put `configuring network options in a separated ticket / PR`, as this is not quite related to `update dependency`.

Reply:
As per `configuring network`, I have a different solution, which sits in the [demo of automation-test](https://github.com/benxgao/substrate-automation-test/tree/master/config), if you are available, would you mind taking a look, or we may have a discussion if needed?